src/authetication/storage.tsx

```diff
@@ -0,0 +1,26 @@
+const setStorage = (name: string, value: string) => {
+  return new Promise((resolve, reject) => {
+    sessionStorage.setItem(name, value);
+    return resolve("OK");
+  });
+};
+
+const deleteStorage = (name: string) => {
+  return new Promise((resolve, reject) => {
+    return resolve(sessionStorage.removeItem(name));
+  });
+};
+
+const resetStorage = () => {
+  return new Promise((resolve, reject) => {
+    return resolve(sessionStorage.clear());
+  });
+};
+
+const getStorage = (item: string) => {
+  return new Promise((resolve, reject) => {
+    return resolve(sessionStorage.getItem(item));
+  });
+};
+
+export { setStorage, deleteStorage, resetStorage, getStorage };
```

Comment:
C'è un motivo per la quale le operazioni sul session storage sono wrappate in delle promise?